### DISCUSSION TITLE
Overwriting SCSS the right way?

### DISCUSSION THREAD

**Discussion by RoneoOrg** [Lednerb/bilberry-hugo-theme#377] [Q&A]
Hi there,

overwriting theme files in Hugo use to be done by editing a copy of these files at the root of the project.

But for Bilberry, SCSS files are updated by [running a npm command](https://github.com/Lednerb/bilberry-hugo-theme#custom-colors-and-fonts) from inside the theme repo, which brings additional complexity at deploy time.

Question: is there a way to overwrite these SCSS files while keeping an unmodified theme repo?
(and without injecting CSS on top, I'm speaking here about leveraging SCSS)

**Comment by igor-baiborodine** [comment]
@RoneoOrg, moved to Discussions.

**Reply by RoneoOrg** [reply]
Thanks @igor-baiborodine!

Related discussion: https://github.com/Lednerb/bilberry-hugo-theme/discussions/339
